dataline-integrations/singer/postgres/source/.dockerignore

```diff
@@ -0,0 +1,2 @@
+build
```

Comment:
Why not make in a allow list instead:
```
*
!Dockefile
!...
```

Reply:
Nice, wasn't familiar with that pattern. Done.